lib/order.rb

```diff
@@ -0,0 +1,53 @@
+require_relative "menu"
+require_relative "printer"
+require_relative "text"
+require_relative "calculator"
+
+class Order
+
+  attr_reader :complete_order, :food_options, :calculator,
+  :total, :print_class, :text_class
+
+  def initialize(food_options = Menu.new.food_options,
```

Comment:
Lots of variables and classes being passed in - could this be simplified?

Reply:
For sure, although the only way that I could see to simplify this would be to break up my order class into two different classes. I guess at the moment it's sort of the main class, which we talked about avoiding last week. Is that what you mean?